okta/resource_okta_group_memberships.go

```diff
@@ -140,21 +151,33 @@ func checkIfGroupHasUsers(ctx context.Context, client *okta.Client, groupId stri
 		groupUsers = append(groupUsers, additionalUsers...)
 	}
 
-	// Create set of users
+	// We need to return false if there isn't any users present. For eventual
```

Comment:
@exitcode0 I think `checkIfGroupHasUsers` needs to exit if users are found here. See my notes in the commit about eventual consistency and the backoff used in create versus no backoff in read.